src/components/shared-settings/settings.ts

```diff
@@ -30,6 +30,18 @@ export const disableOpenNewTabInBackgroundSettings = createNewSettings<boolean>(
     },
 )
 
+export const renderInShadowRootSettings = createNewSettings<boolean>(
+    'render in shadow root',
+    /**
+     * ? In WKWebview, the web extension polyfill is not ready for it.
+     */
+    webpackEnv.target === 'WKWebview' ? false : true,
```

Comment:
`webpackEnv.target !== 'WKWebview'`

Reply:
it named `renderInShadowRootSettings` internally. So `true` will use ShadowRoot. `false` won't

Reply:
Yeah. The statement can be shorter.